docs/remote/containers.md

```diff
@@ -231,6 +235,51 @@ Selecting the **Remote-Containers: Add Development Container Configuration Files
 
 To learn more about creating `devcontainer.json` files, see [Create a Development Container](/docs/remote/create-dev-container.md).
 
+## Dev container features
+
+Dev container features provide a smooth path for customizing your container definitions.
+
+When you use **Remote-Containers: Add Development Container Configuration Files**, you're presented a list of scripts to customize the existing dev container configurations, such as installing Git or the Azure CLI:
+
+![Dev container features in Command Palette](images/containers/container-features.png)
+
+When you rebuild and reopen in your container, the features you selected will be available in your `devcontainer.json`:
+
+```json
+"features": {
+		"github-cli": "latest"
+	}
+```
+
+You'll get IntelliSense when editing the `"features"` property in the `devcontainer.json` directly:
+
+![Intellisense when modifying terraform feature](images/containers/features-intellisense.png)
+
+The features are sourced from the [script library](https://github.com/microsoft/vscode-dev-containers/tree/main/script-library/docs) in the vscode-dev-containers repo.
+
+The **Remote-Containers: Configure Container Features** command allows you to update an existing configuration.
+
+    > **Note:** Features support for GitHub Codespaces is coming soon.
+
+## Pre-building dev container images
+
+You can pre-build images with the tools you need rather than newly creating and building images for each project. Using pre-built images can be simpler and allows you to pin to a specific version of tools, avoiding potential breaks. Pre-building is especially valuable in CI processes.
```

Comment:
I think this sentence works but was thinking about phrasing it along the lines of "rather than creating and building the image each time you create a container for a project." I think we can drive home a little stronger that you're saving time whenever you use this definition!

Reply:
I like that suggestion! Updated to what you have in quotes.